test/fresh.js

```diff
@@ -107,18 +107,73 @@ describe('fresh(reqHeaders, resHeaders)', function () {
     describe('with invalid If-Modified-Since date', function () {
       it('should be stale', function () {
         var reqHeaders = { 'if-modified-since': 'foo' }
-        var resHeaders = { 'modified-since': 'Sat, 01 Jan 2000 00:00:00 GMT' }
+        var resHeaders = { 'last-modified': 'Sat, 01 Jan 2000 00:00:00 GMT' }
         assert.ok(!fresh(reqHeaders, resHeaders))
       })
     })
 
-    describe('with invalid Modified-Since date', function () {
+    describe('with invalid Last-Modified date', function () {
       it('should be stale', function () {
         var reqHeaders = { 'if-modified-since': 'Sat, 01 Jan 2000 00:00:00 GMT' }
-        var resHeaders = { 'modified-since': 'foo' }
+        var resHeaders = { 'last-modified': 'foo' }
+        assert.ok(!fresh(reqHeaders, resHeaders))
+      })
+    })
+
+    describe('with earliest possible If-Modified-Since date', function () {
+      it('should be stale', function () {
+        var reqHeaders = { 'if-modified-since': 'Thu, 01 Jan 1970 00:00:00 GMT' }
+        var resHeaders = { 'last-modified': 'Sat, 01 Jan 2000 00:00:00 GMT' }
         assert.ok(!fresh(reqHeaders, resHeaders))
       })
     })
+
+    describe('with earliest possible Last-Modified date', function () {
+      it('should be fresh', function () {
+        var reqHeaders = { 'if-modified-since': 'Sat, 01 Jan 2000 00:00:00 GMT' }
+        var resHeaders = { 'last-modified': 'Thu, 01 Jan 1970 00:00:00 GMT' }
+        assert.ok(fresh(reqHeaders, resHeaders))
+      })
+    })
+
+    describe('with earliest possible Last-Modified and If-Modified-Since date', function () {
+      it('should be fresh', function () {
+        var reqHeaders = { 'if-modified-since': 'Thu, 01 Jan 1970 00:00:00 GMT' }
+        var resHeaders = { 'last-modified': 'Thu, 01 Jan 1970 00:00:00 GMT' }
+        assert.ok(fresh(reqHeaders, resHeaders))
+      })
+    })
+
+    describe('when Date.parse was monkey patched', function () {
```

Comment:
A devDep is just fine, as they are available during the testing. If you want to just not have any tests, that's OK too, but if you wanted to make some patching tests, I'd say to either (a) directly use datejs in them or (b) make the mocking patches actually function like the datejs does (which afaik these are different from what datejs returns).